src/schema.graphql

```diff
@@ -9,24 +9,24 @@ input IssueInput {
 }
 
 type Query {
-  info: String!,
+  info: String!
   divisions: [Division!]!
   division(where: DivisionWhereUniqueInput!): Division
   dailyReports: [DailyReport!]!
   dailyReport(where: DailyReportWhereUniqueInput!): DailyReport
   issues: [Issue!]!
   issue(where: IssueWhereUniqueInput!): Issue
   me: User!
+  currentUserReports: [DailyReport!]!
```

Comment:
If you have better naming for this query, let me know 🙇‍♂️

Reply:
@queq1890 The naming is should be only `userReports` 
Why ? 
The purpose is reuse query.
If I'm the current user, I can easily get other users by this query that don't need create a new query.

Reply:
@phuongthuan I'm gonna rename it. Thx!